app/models/utils/load_testing/sms_submission_load_test.rb

```diff
@@ -0,0 +1,108 @@
+# frozen_string_literal: true
+
+module Utils
+  module LoadTesting
+    # Builds a JMeter test plan for submitting a lot of SMS form submissions.
+    #
+    # This test building process should be run on the server that is being targeted
+    # for the test because it will query the database while building the sample submission
+    # data (i.e. the message body for the SMS submission) for the test.
+    #
+    # The actual JMeter test will be run on a different server that does not have NEMO installed.
+    # It will just know how to make a bunch of HTTP requests based on the options and data we give it.
+    class SmsSubmissionLoadTest < LoadTest
+      # Required `options`:
+      #   user_id: User doing the submitting
+      #   form_id: The ID of the form being submitted to.
+      #
+      # The test currently knows how to handle the following question types:
+      #   text, long_text, integer, counter, decimal, select_one,
+      #   select_multiple, datetime, date, time
+      #
+      # * The submitting user must have a phone number and SMS auth code
+      # * The Twilio adapter must be configured appropriately on the target
```

Comment:
Not sure if we can use Twilio because we would have to sign each message correctly for it to work. We could do something like:

```
validator = Twilio::Util::RequestValidator.new("xyz")
validator.build_signature_for("#{configatron.url.protocol}://#{configatron.url.host}/foo", {"From" => "+123456776", "To" => "+1234567890", "Body" => "abc aldflat"}) 
```

for each message and store that in the CSV file. It seems fraught though. Another option would be to use the frontline cloud adapter which doesn't do any inbound validation.